gen_card_list.rb

```diff
@@ -8,6 +8,7 @@
 
 require 'nokogiri'
 require 'json'
+require "../util.rb"
```

Comment:
```suggestion
require_relative "../util.rb"
```
これはローカルのライブラリなので、`require_relative`にしたいですがどうでしょうか

Reply:
`require_relative`, 使ったことなかったです。
確かに、ローカルであることが分かりやすいほうが良さそうですね。

手元で試してみたところ、
`require_relative "../util"` だとプロジェクトルートのひとつ上のディレクトリの`util`を探しに行ってしまうので、
`require_relative "util"` でいいようです。

Reply:
9bab2df のコミットで修正しました！

Reply:
ああ、なるほど確かにここは `..` は不要そうですね。確認ありがとうございます！